run_travis_tests.sh

```diff
@@ -27,6 +27,13 @@ git clone https://github.com/mozilla/build-mozharness mozharness
 echo "Installing tools..."
 git clone https://github.com/mozilla/build-tools tools
 
+echo "Installing repo..."
+curl https://storage.googleapis.com/git-repo-downloads/repo > ./repo && chmod a+x ./repo
+echo "Configuring Git for Repo ..."
+git config --global user.email "mozilla-b2g@github.com"
+git config --global user.name "Mozilla B2G"
+git config --global color.ui false
```

Comment:
We need to set that otherwise |repo init| will first fails because of non configured name/email and then hangs on checking colors